backend/oasst_backend/models/message_revision.py

```diff
@@ -23,3 +24,5 @@ class MessageRevision(SQLModel, table=True):
     created_date: Optional[datetime] = Field(
         sa_column=sa.Column(sa.DateTime(timezone=True), nullable=True, server_default=sa.func.current_timestamp())
     )
+
+    _user_is_author: Optional[bool] = PrivateAttr(default=None)
```

Comment:
Does this now work without work-around? e.g. see https://github.com/tiangolo/sqlmodel/issues/149?

In the Message ORM class in message.py I had to insert:
```
    def __new__(cls, *args: Any, **kwargs: Any):
        new_object = super().__new__(cls, *args, **kwargs)
        # temporary fix until https://github.com/tiangolo/sqlmodel/issues/149 gets merged
        if not hasattr(new_object, "_user_emojis"):
            new_object._init_private_attributes()
        return new_object
    ```

Reply:
Nope, not fixed. It only works because I set `_user_is_author` immediately after fetching the revisions, just like the `_user_is_author` field in the `message` table.